lib/instrument/index.js

```diff
@@ -0,0 +1,176 @@
+// Adapted from https://github.com/YueChen-C/py-ios-device
+
+import {BaseServiceSocket} from '../base-service';
+import {DTXMessage, FLAG_TYPES} from './headers';
+import {DTXEncoder} from './transformer/dtx-encode';
+import {DTXDecoder} from './transformer/dtx-decode';
+import events from 'events';
+import _ from 'lodash';
+import {waitForCondition} from 'asyncbox';
+const CHECK_FREQ_MS = 500;
+const WAIT_REPLY_TIME = 10000;
```

Comment:
I usually add unit suffix to each constant name to avoid confusion, like it was done in the previous one (`_MS`)